src/table/tableview/table-view.component.js

```diff
@@ -3,6 +3,7 @@ angular.module('patternfly.table').component('pfTableView', {
     config: '<?',
     dtOptions: '<?',
     colummns: '<',
+    colvis: '<',
```

Comment:
colvis is not intuitive as to what it is used for. Please spell it out as column-visibility